src/Common/src/TypeSystem/Interop/MethodDesc.Interop.cs

```diff
@@ -28,6 +28,47 @@ public virtual PInvokeMetadata GetPInvokeMethodMetadata()
         {
             return default(PInvokeMetadata);
         }
+
+        /// <summary>
+        /// If <see cref="IsPInvoke"/> is true, retrieves the parameters of the platform invoke.
```

Comment:
This comment is not correct - parameter metadata exists for non-pinvokes too. All you need is e.g. a method that has default parameters (which is why I suggested dropping the `PInvoke` part of the identifiers you're adding in this file - they're not PInvoke specific, although most of them are for PInvokes).